flow/scripts/variables.yaml

```diff
@@ -108,7 +114,7 @@ PROCESS:
     Technology node or process in use.
 CORNER:
   description: >
-    Library to select based on corner BC/TC/WC.
+    PVT corner library selection. Only available for ASAP7, GF55 and GF180 PDKs.
```

Comment:
gf55 isn't a public platform so please remove it.  That should fix the scan error

Reply:
removed and pushed up a squashed commit